services/src/main/java/org/fao/geonet/api/OpenApiController.java

```diff
@@ -159,14 +161,15 @@ && isPathToMatch(operationPath)) {
                     // default allowed requestmethods
                     if (requestMethods.isEmpty())
                         requestMethods = this.getDefaultAllowedHttpMethods();
-                    calculatePath(handlerMethod, operationPath, requestMethods);
+                    calculatePath(handlerMethod, operationPath, requestMethods, locale);
 //                } else {
 //                    System.out.println("API path ignored: " + operationPath);
                 }
             }
         }
     }
 
+
```

Comment:
Add `@Override` to the method.
```suggestion
    @Override
```